core/oa_hash_map.h

```diff
@@ -341,17 +350,36 @@ class OAHashMap {
 		return it;
 	}
 
-	OAHashMap(const OAHashMap &) = delete; // Delete the copy constructor so we don't get unexpected copies and dangling pointers.
-	OAHashMap &operator=(const OAHashMap &) = delete; // Same for assignment operator.
+	OAHashMap(const OAHashMap &p_other) {
+		capacity = 0;
```

Comment:
This one seems redundant as it's already initialized to `0` on declaration.

Reply:
Yes, I added it to make it explicit what is going to happen. I can remove it if you still think it's useless though.

Reply:
Solved